views/blogs/index.ejs

```diff
@@ -28,5 +28,8 @@
         <% }); %>
     </div>
 </div>
+<footer>
 
-<% include ../partials/footer %>
\ No newline at end of file
+    <% include ../partials/footer %>
+
+</footer>
```

Comment:
Same for this. Transfer them all to the footer.ejs file.

Reply:
@thebishaldeb Actually i did that on purpose, so that I can able to make changes for footer component, Using footer tag.

Reply:
@thebishaldeb ohkk, I have done, what you said, could you review now #42 and merge it.

Reply:
> purpose

Yeah, but the code becomes redundant due to copying the same code everywhere everytime we make a new page. If we keep the footer tag in the footer file, we can just include the footer file at the end of each page.

Reply:
@thebishaldeb, Yeah I did that, could you see #42